README.md

```diff
@@ -1,106 +1,125 @@
-*Looking for the [Dwellingly App front end?](https://github.com/codeforpdx/dwellingly-app)*
+_Looking for the [Dwellingly App front end?](https://github.com/codeforpdx/dwellingly-app)_
 
 ## Dwellingly App Backend
 
 Set up Dwelling Flask Testing Backend (for the first time)
-NOTE: Database is SQLite3 via SQLAlchemy 
+NOTE: Database is SQLite3 via SQLAlchemy
 
-+ Github Repo: { https://github.com/codeforpdx/dwellinglybackend }
-+ Live Server: {https://dwellinglyapi.herokuapp.com/} 
+- Github Repo: { https://github.com/codeforpdx/dwellinglybackend }
+- Live Server: {https://dwellinglyapi.herokuapp.com/}
 
 ### To Start Server
 
 1. Clone the repo (`git clone https://github.com/codeforpdx/dwellinglybackend.git`)
 2. Install Python ( https://realpython.com/installing-python/ )
 3. Install pipenv: `pip3 install --user pipenv`
 4. Seed the database
-    - Run: `python seed_db.py`
-      - To re-seed the database from scratch, delete data.db before running the script
-    - Look for the file data.db to be created in the root directory
 
-5. Start the server using the flask environment (required every time the project is re-opened):
-    - Run: `pipenv run flask run`
-    - Run + restart the server on changes:  `pipenv run flask run --reload`
+   - Run: `python seed_db.py`
+     - To re-seed the database from scratch, delete data.db before running the script
+   - Look for the file data.db to be created in the root directory
+
+5. Run `pipenv install`
+6. Start the server using the flask environment (required every time the project is re-opened):
+   - Run: `pipenv run flask run`
+   - Run + restart the server on changes: `pipenv run flask run --reload`
 
 Queries can be made with the Postman Collection link ( https://www.getpostman.com/collections/a86a292798c7895425e2 )
 
+### Note For Windows Users
+
+Python does not come by default for Windows users. Sometimes the PATH variable in Windows will point to the wrong path and you will have trouble running the `python` and `pipenv` commands. If you don't have Python installed then follow these steps. If the steps listed above did not work for you, try the following.
+
+1. Clone the repo (`git clone https://github.com/codeforpdx/dwellinglybackend.git`)
+2. Hit the Windows key on your keyboard and type `Microsoft Store`. Click on Microsoft store and search for Python. Download Python from the official Microsoft store. When you are installing **make sure you tick the Add to PATH checkbox [ ]** If you don't do this, it could result in you being unable to run the python or pip / pipenv commands.
+3. Once Python is installed, run `pip3 install --user pipenv` If this command doesn't work, you can also try running `python -m pip` followed by running `python -m pip install --user pipenv`
```

Comment:
Missing a period after the first sentence. `python -m pip` just prints out the usage instructions; `python -m pip install --user pipenv` should be enough. This step also applies for non-Windows users if for some reason their pip binary is a different version from what is expected.

Reply:
Ok, thanks for the review. Updated again